### PR TITLE
CB-2660: Add set property mutation to resource manager

### PR DESCRIPTION
**Pull request by oksana-cherniavskaia**

### CODE REVIEW

File: webapp/packages/core-sdk/src/queries/resource-manager/setResourceProperty.gql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+mutation setResourceProperty(
```
Comment (serge-rider) [inline webapp/packages/core-sdk/src/queries/resource-manager/setResourceProperty.gql:1]: Don't do it (don't add files .gql) it will be added by FE team if needed.

File: server/bundles/io.cloudbeaver.service.rm/src/io/cloudbeaver/service/rm/DBWServiceRM.java [hunk 2]
```diff
@@ -58,6 +58,27 @@ RMResource[] listResources(
...
+     *
+     * @param webSession    the web session
+     * @param projectId     the project id
+     * @param resourcePath  the resource path
+     * @param propertyName  the property name
+     * @param propertyValue the property value
+     * @return the resource property
+     * @throws DBException the db exception
+     */
+    @WebProjectAction(
```
Comment (serge-rider) [inline server/bundles/io.cloudbeaver.service.rm/src/io/cloudbeaver/service/rm/DBWServiceRM.java]: Keep it in single line

File: server/bundles/io.cloudbeaver.service.rm/src/io/cloudbeaver/service/rm/impl/WebServiceRM.java [hunk 3]
```diff
@@ -92,7 +130,11 @@ public String readResourceAsString(@NotNull WebSession webSession, @NotNull Stri
     }
 
     @Override
-    public String createResource(@NotNull WebSession webSession, @NotNull String projectId, @NotNull String resourcePath, boolean isFolder) throws DBException {
+    public String createResource(@NotNull WebSession webSession,
```
Comment (serge-rider) [inline server/bundles/io.cloudbeaver.service.rm/src/io/cloudbeaver/service/rm/impl/WebServiceRM.java:133]: Don't do unrelated code reformat.

Comment (oksana-cherniavskaia) [reply server/bundles/io.cloudbeaver.service.rm/src/io/cloudbeaver/service/rm/impl/WebServiceRM.java:133]: will do next time

File: webapp/packages/core-sdk/src/queries/resource-manager/setResourceProperty.gql (hunk 1 above)
Comment (oksana-cherniavskaia) [reply webapp/packages/core-sdk/src/queries/resource-manager/setResourceProperty.gql:1]: Will ask FE team is it needed.

File: server/bundles/io.cloudbeaver.service.rm/src/io/cloudbeaver/service/rm/DBWServiceRM.java (hunk 2 above)
Comment (oksana-cherniavskaia) [reply server/bundles/io.cloudbeaver.service.rm/src/io/cloudbeaver/service/rm/DBWServiceRM.java]: fixed

File: webapp/packages/core-sdk/src/queries/resource-manager/setResourceProperty.gql (hunk 1 above)
Comment (oksana-cherniavskaia) [reply webapp/packages/core-sdk/src/queries/resource-manager/setResourceProperty.gql:1]: deleted
